composer.lock

```diff
@@ -2685,16 +2752,16 @@
         },
         {
             "name": "phpoffice/phpspreadsheet",
-            "version": "1.4.1",
+            "version": "1.5.2",
```

Comment:
Could you please have a look at the composer.lock as well. Looks as if you executed a general composer update and all available package updates were pulled in. I like to do that in a seperate PR, to have the chance to review the updated packages (if they are directly used by Kimai) and to see if they changed anything important. 
So I'd like to see only changes in composer.lock which are dedicated to the 2FA packages and not stuff like Phpoffice or Doctrine.

Reply:
The simplest approach should be: revert composer.json / composer.lock / symfony.lock to their original state, then execute a `composer install` => the new packages should be replaced by the old ones. Then execute a `composer require scheb/two-factor-bundle`.